src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -1475,6 +1476,46 @@ void valid() {
                 testValid("NISTHypothesis.valid: there should be exactly 1 hypothesis");
             }
         }
+
+        @Nested
+        class EntityClusterRequiredHandle {
+            @Test
+            // No handle property on entity cluster in hypothesis
+            void invalidNoHandle() {
+                Resource newEntity = makeEntity(model, getEntityUri(), system);
+                addType(newEntity, SeedlingOntology.Person);
```

Comment:
Type added below. Please remove this line

Reply:
Updated.